R/survival.R

```diff
@@ -683,11 +681,10 @@ get_migratory_survival_rates <- function(year, month,
 
   bay_delta_migration_surv <- mean(c(0.43, 0.46, 0.26, 0.25, 0.39)) # Bay.S Chipps island to bay
```

Comment:
this might be a parameter needed for sensitivity analysis, @ErinCain plz check with J+A